service_worker/index.ts

```diff
@@ -25,6 +25,12 @@ declare let self: ServiceWorkerGlobalScope;
 
 precacheAndRoute(self.__WB_MANIFEST);
 
+// Don't waste the user's local storage on Easter Eggs.
```

Comment:
nit: I'd opt for simplicity and wouldn't worry about this exception

Reply:
I think I would prefer to keep this in. `/easter-eggs/001` is over 500K, whereas the entirety of the rest of the app bundle is only around 60K. I think that's a substantial enough difference to have a few extra lines.